pkg/lockfile/parse-gradle-lock.go

```diff
@@ -54,16 +63,27 @@ func ParseGradleLock(pathToLockfile string) ([]PackageDetails, error) {
 
 		pkg, err := parseToGradlePackageDetail(lockLine)
 		if err != nil {
-			fmt.Fprintf(os.Stderr, "failed to parse lockline: %s\n", err.Error())
+			fmt.Fprintf(os.Stderr, "failed to extract from lockline: %s\n", err.Error())
 			continue
 		}
 
 		pkgs = append(pkgs, pkg)
 	}
 
 	if err := scanner.Err(); err != nil {
-		return []PackageDetails{}, fmt.Errorf("failed to read %s: %w", pathToLockfile, err)
+		return []PackageDetails{}, fmt.Errorf("failed to read: %w", err)
 	}
 
 	return pkgs, nil
 }
+
+var _ Extractor = GradleLockExtractor{}
+
+//nolint:gochecknoinits
+func init() {
+	registerExtractor("gradle.lockfile", GradleLockExtractor{})
```

Comment:
Would there be any bad side effects if we register the extractor twice? e.g.:
```
	registerExtractor("gradle.lockfile", GradleLockExtractor{})
	registerExtractor("buildscript-gradle.lockfile", GradleLockExtractor{})
```

Reply:
Yes, that causes the original problem with extractor names - see https://github.com/google/osv-scanner/pull/451/commits/c9812149f467861aa09252e7e240181708dea46c

Reply:
Ah I see what you mean now, the parser name being returned won't be consistent. Let's go with what we have now then (i.e. only registering gradle.lockfile). 

This is consistent with us keeping the extractor name separated from what it can extract.